skyportal/handlers/api/annotation.py

```diff
@@ -98,17 +158,18 @@ def post(self):
         if not group_ids:
             groups = self.current_user.accessible_groups
         else:
-            groups = Group.get_if_accessible_by(
-                group_ids, self.current_user, raise_if_none=True
-            )
-
+            try:
+                groups = Group.get_if_accessible_by(
+                    group_ids, self.current_user, raise_if_none=True
+                )
+            except AccessError:
+                return self.error('Could not find any accessible groups.')
```

Comment:
Appropriate HTTP status

Reply:
This would be a 405?

Reply:
403, I think.

We may be able to handle this generically, as I asked @acrellin about, but I don't think it currently is.

I made this related PR: https://github.com/cesium-ml/baselayer/pull/253/files which is not quite right yet, but should cover these use cases eventually.

Reply:
So looking at this PR I think maybe a 401 is more appropriate?

Reply:
401 is when the user is unauthenticated